### PR TITLE
Create a file for chartek chicago

### LINKED ISSUE #43: chartek's recommendations for Chicago, IL
I would like to add a file for Chicago, IL to this repository.

I will follow the GitHub flow:

- [x]  Create a branch
- [x]  Add my file
- [x]  Commit my changes
- [x]  Open a pull request
- [ ]  Collaborate with my friends
- [ ]  Get a 👍
- [ ]  Merge by Pull Request

### PR DESCRIPTION
**Pull request by chartek**
This work will close #43.

## Things left to do

- [ ] Add a list of places to eat
- [ ] Add a list of fun things to do

### CODE REVIEW

File: chartek-chicago.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Recommendations in Chicago, IL
+
+### Places to eat
```
Comment (chartek) [inline chartek-chicago.md]: What are your favorite places to eat? What are your favorite types of foods?

Comment (ehertlein) [inline chartek-chicago.md]: dont forget lavash!

File: chartek-chicago.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Recommendations in Chicago, IL
```
Comment (aschulman84) [inline chartek-chicago.md]: asdf

Comment (chartek) [inline chartek-chicago.md]: Sounds good!

**Pull request merged**